Felienne/hedy
# [FIX] Shows line numbers on view shared / submitted programs

**Description**
In this PR we fix an issue where line numbers where not shown on shared or submitted programs. By verifying if the editor with the `readOnly` attribute is actually the main editor we add the `showGutter: true` attribute to show line numbers.

**Fix for**
This PR fixes #1789.

**How to test**
View either a shared or submitted program and notice that the line numbers are shown in the editor.
